src/cpl.rs

```diff
@@ -59,9 +83,10 @@ impl CslStringList {
         Ok(())
     }
 
-    /// Adds the string `value` to the list.
+    /// Adds a copy of the string slice `value` to the list.
     ///
-    /// Returns `Ok<()>` on success, `Err<GdalError>` if `value` cannot be converted to a C string.
+    /// Returns `Ok<()>` on success, `Err<GdalError>` if `value` cannot be converted to a C string,
```

Comment:
I know these are old, but we should probably take the chance to fix them to say `Ok(())` and `Err(GdalError::Whatever)`.